torch/csrc/jit/codegen/cuda/scheduler/utils.cpp

```diff
@@ -31,23 +34,23 @@ size_t mergeReduction(TensorView* tv) {
       num_merged++;
     }
   }
-  if (prev_i == 0) {
-    tv->reorder({{prev_i, -1}});
+  if (prev_i != 0) {
```

Comment:
This puts the merged reduction domain on the left. If `mergeNonReduction` is called next it will put the merged non-reduction to the left. Just curious is this trying to order the final axes as this ?
```
[merged-non-reduction-axes , merge-reduction-axes , dont-touch-axes]
```

Reply:
It just makes it more consistent with the other merge function. Anything that gets merged will be moved to the first position of the resulting tv. If you call this than the iter merge it will be `[iter, reduction, don't merge]` if you do it in the reverse order it will be `[reduction, iter, don't merge]`.

Reply:
I see. Thanks for the clarification.